README.md

```diff
@@ -7,9 +7,61 @@
 An [Apache 2.0](https://github.com/allenai/allennlp/blob/master/LICENSE) NLP research library, built on PyTorch,
 for developing state-of-the-art deep learning models on a wide variety of linguistic tasks.
 
-## Quickstart
+## Running AllenNLP
+
+The preferred way to install AllenNLP is via `pip`.
+
+### Setting up a virtual environment
+
+[Conda](https://conda.io/) can be used set up a virtual environment
+with the version of Python required for AllenNLP and in which you can
+sandbox its dependencies:
+
+1.  [Download and install Conda](https://conda.io/docs/download.html).
+
+2.  Create a Conda environment with Python 3.6
+
+    ```
+    conda create -n allennlp python=3.6
+    ```
+
+3.  Activate the Conda environment.  (You will need to activate the Conda environment in each terminal in which you want to use AllenNLP.
+
+    ```
+    source activate allennlp
+    ```
+
+4. Install AllenNLP in your environment.
+
+### Installing via pip
+
+The preferred way to install AllenNLP into your environment is via `pip`.
+
+1.  Install AllenNLP.
+
+    ```
+    pip install allennlp
+    ```
+
+2. Visit http://pytorch.org/ and install the PyTorch 0.3.1 package for your system.
+
+3. Download the spacy models. 
+
+    ```
+    python -m spacy download en_core_web_sm
+    ```
+
+That's it! You're now ready to build and train AllenNLP models.
+AllenNLP installs a script when you install the python package, meaning you can run allennlp commands just by typing `allennlp` into a terminal.
+Now you can do any of the following:
+
+* Run a model on example sentences with `allennlp predict`.
```

Comment:
I'd probably just remove this list.  The list is actually a whole lot longer than this; including the whole list is probably not a good idea, and I'm not sure why you picked these two.